tests/ci/cdk/cdk/ssm/m1_tests_ssm_document.yaml

```
@@ -0,0 +1,27 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+---
+schemaVersion: '2.2'
+description: aws-lc:bmFrameworkEc2Benchmark
+mainSteps:
+  - action: aws:runShellScript
+    name: runShellScript
+    inputs:
+      timeoutSeconds: '7200'
+      runCommand:
+        - eval $(/opt/homebrew/bin/brew shellenv)
+        - sudo su
+        - BUILD_ROOT=$(mktemp -d)
+        - cd ${BUILD_ROOT}
+        - export HOME="/var/root"
+        - trap "rm -rf ${BUILD_ROOT}/aws-lc-pr" EXIT
```

```suggestion
        - trap "rm -rf ${BUILD_ROOT}" EXIT
```
This leaks the temp folder and in a few years we might run out of folder ids.

---

good catch, fixed :)